doc/02_user/getting_started.rst

```diff
@@ -3,4 +3,16 @@
 Getting Started with Ibex
 =========================
 
+The Ibex repository contains all the RTL needed to simulate and synthesize an Ibex core.
+`FuseSoC <https://github.com/olofk/fusesoc>`_ core files list the RTL files required to build Ibex (see :file:`ibex_core.core`).
+The core itself is contained in the :file:`rtl/` directory, though it utilizes some primitives found in the :file:`vendor/lowrisc_ip/` directory.
+These primitives come from the `OpenTitan <https://github.com/lowrisc/opentitan>`_ project but are copied into the Ibex repository so the RTL has no external dependencies.
+You may wish to replace these primitives with your own and some are only required for specific configurations.
+See :ref:`integration-prims` for more information.
 
+There are several paths to follow depending on what you wish to accomplish:
+
+ * See :ref:`examples` for a basic simulation setup running the core in isolation and a simple FPGA system.
```

Comment:
Obviously, the simple FPGA system lives now the demo repo. But it's probably fine to keep the pointer here.